Alina Firsova/flex/style.css

```diff
@@ -0,0 +1,190 @@
+body, html {
+    height: 100%;
+    margin: 0;
+}
+
+@media screen and (min-width: 615px) {
+    .main-container { 
+        width: 615px;
+        height: 410px;
+        
+        }
+        
+        
+    }
+
+    @media screen and (max-width: 615px) {
+        .main-container { 
+            height: 615px;
+            
+            }
+            
+            
+        }

```

Comment:
@AlinaFirsova you can set styles for main-container by defult (for small screens for ex.) and then add media only for bigger cases
`
.main-container { 
    height: 615px;
}
@media screen and (min-width: 615px) {
    .main-container { 
        width: 615px;
        height: 410px;    
    }        
}

`